tools/generator/Tests/Unit-Tests/CodeGeneratorExpectedResults/JavaInterop1/WriteInterfaceDefaultMethod.txt

```diff
@@ -0,0 +1,34 @@
+// Metadata.xml XPath interface reference: path="/api/package[@name='java.code']/interface[@name='IMyInterface']"
+[Register ("java/code/IMyInterface", "", "java.code.IMyInterfaceInvoker")]
```

Comment:
Where should `IMyInterfaceInvoker` be declared?  Is that a separate unit test?  *Should* it be separate?  I don't see any class definitions of `*Invoker` types within this PR.

Reply:
This PR does not make any changes to `*Invoker` types, so there are no additional unit tests for them.

Reply:
This PR *implicitly* "makes changes" to `*Invoker` types, as it raises the question: do the `*Invoker` types override default interface methods?

They don't *need* to, but without actually seeing the codegen outputs it's hard to *verify* that it isn't actually doing so.

Reply:
Added a DIM `*Invoker` unit test.